src/requela/grammar.lark

```diff
@@ -108,6 +109,7 @@ EMPTY_LITERAL.2: "empty()"
 
 # Property name
 PROP: /[a-zA-Z_][\w\-\.]*/  # Property names: letters, numbers, underscore, dash, dot
+UUID.3: /[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/ # increase priority to ensure that UUIDs are matched before generic PROP
```

Comment:
Please move such terminal together with the others with priority 3. 

You should also add also a method to treat such type into the transformer for consistency with other terminals (even if it won't be converted to UUID type)